projects/components/src/overlay/sheet/sheet-overlay.component.ts

```diff
@@ -81,6 +87,7 @@ export class SheetOverlayComponent {
     this.hasDefaultPadding = sheetConfig.hasDefaultPadding ?? true;
     this.showHeader = sheetConfig.showHeader === true;
     this.sheetTitle = sheetConfig.title === undefined ? '' : sheetConfig.title;
+    this.isSheetTitleAString = typeof this.sheetTitle === 'string';
```

Comment:
would be cleaner to check the inverse with `instanceof TemplateRef`

Reply:
Not sure, I did the same thing at many places, so continued with the same :)

Reply:
we can avoid this :)